### DISCUSSION TITLE
GO on Sourceforge

### DISCUSSION THREAD

**Discussion by midimusic** [GrandOrgue/grandorgue#1532] [General]
I happened to have a look at the old GO website on Sourceforge today and have noted that it has not been updated since 2019.

However I also noted that there had been 189 downloads of 64 bit windows GO version  0.3.1.2330..... this week from this site. Has the news that GO is now on github not yet reached all these people? Or is there some other reason?

csw900

**Comment by ClarkeBrunt** [comment]
and by now (end of 18th May 2023), Sourceforge says "266 downloads this week". Does github offer any similar stats on downloads?

There is no shortage of information on the Sourceforge page that development is now on github: a link to the github page saying "As of 2021-09-05, this project can be found here...", and then "Official development of GrandOrgue has moved to GitHub, please visit...", so not sure what else can be done.

I wonder if these are downloads by real people, or maybe just 'bots' sucking up the supposedly latest version and then offering it on other sites. Searching for 'grandorgue download" in a search engine readily gave several sites offering the 0.3... version - not sure why anyone would use these sites, but I expect people do, and these other sites won't include the information that that development is now on github.

**Reply by rousseldenis** [reply]
@larspalo 

Even if we need to let old files in place, is there a mean to explicit say something that those files are deprecated and also remove the green 'Download the latest version' button?

**Reply by larspalo** [reply]
@rousseldenis I'll log into SourceForge and see what can be done. It should certainly be possible to edit the README.rest.txt file to state that all of the files on SourceForge are now obsolete and that current builds are available here on GitHub instead.

**Reply by rousseldenis** [reply]
@larspalo Yes, or create an 'archive' directory to put all files. That is more radical but should avoid downloads

**Reply by larspalo** [reply]
@rousseldenis The most radical approach would be to delete all the files, but for now I think it's best to leave them with a proper note...

**Reply by larspalo** [reply]
@rousseldenis @oleg68 It seems that it is possible to setup an automatic release import to the GrandOrgue page on SourceForge from the project releases here on GitHub. Do you think that would be useful?

**Reply by rousseldenis** [reply]
I think Sourceforge project has a great history on search engines. So, it could be great to have releases deployed there too.

**Reply by larspalo** [reply]
It's done by a webhook that I can add here on GitHub that will post any release done here also to Sourceforge, so it won't be any real maintenance burden for us. I've no idea how it will actually look like on the Sourceforge page until we've tried it, but I've at least changed the README.rest.txt to state also there that the project has moved here and also updated the wiki link and support help link to point to the current counterparts here.

**Reply by kerkovits** [reply]
/resources/GrandOrgue.appdata.xml.in l 32. is also obsolete, and points all GNOME users to the old website. There may be other similar instances, as well.

**Reply by larspalo** [reply]
@kerkovits I've added a PR with updates to the appdata file.

**Reply by larspalo** [reply]
@rousseldenis @oleg68 I've added a webhook that will post any release done here on GitHub to the old Sourceforge page too. We can see how that looks and if it's not deemed to be beneficial, we can easily remove the webhook integration anytime.
